src/licensedcode/data/licenses/copyheart.LICENSE

```diff
@@ -0,0 +1,53 @@
+? Copying is an act of love. Please copy and share.
```

Comment:
This is full of mojibake  https://en.wikipedia.org/wiki/Mojibake ... Can you find a way to get a clean plain text for this? but then.... the text is really only that first line, the rest is not: therefore just get rid of the rest all right IMHO

Reply:
Didn't get what you meant. Can you please explain

Reply:
Have you seen all the junk characters in this file?

Reply:
yeah. @pombredanne so what am i supposed to do with those characters.

Reply:
You need to find or craft a clean text-only version of this

Reply:
there are hearts in the text. what am i supposed to do with those?